package.json

```diff
@@ -27,13 +27,14 @@
     "chai": "^2.2.0",
     "grunt": "^0.4.5",
     "grunt-cafe-mocha": "^0.1.13",
+    "grunt-cli": "^0.1.13",
     "grunt-contrib-clean": "^0.6.0",
     "grunt-env": "^0.4.4",
     "grunt-exec": "^0.4.6",
     "load-grunt-tasks": "^3.1.0",
     "mocha": "^2.2.1",
     "nock": "^1.4.0",
-    "time-grunt": "^1.1.0",
-    "rewire": "^2.3.1"
+    "rewire": "^2.3.1",
+    "time-grunt": "^1.1.0"
```

Comment:
When I did `npm install grunt-cli --save-dev` it also alphabetized the devDeps list. Figured that'd be fine but I could also remove that if you want.